src/test/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtilsUnitTest.java

```diff
@@ -93,6 +97,48 @@ public void testHomozygousAlleleList() throws Exception {
         Assert.assertEquals(alleles, Arrays.asList(T, T));
     }
 
+    final static private Locatable START_AT_1 = new SimpleInterval("1",1,1);
+    final static private Locatable START_AT_2 = new SimpleInterval("1",2,2);
+
+    @DataProvider
```

Comment:
Can you refactor these tests to emit `VariantContext` objects rather than these position and allele lists?

Reply:
done